pkg/cmd/auth/status/status.go

```diff
@@ -141,11 +141,13 @@ func NewCmdStatus(f *cmdutil.Factory, runF func(*StatusOptions) error) *cobra.Co
 		Use:   "status",
 		Args:  cobra.ExactArgs(0),
 		Short: "View all accounts and authentication status",
-		Long: heredoc.Doc(`Verifies and displays information about your authentication state.
+		Long: heredoc.Docf(`
+			Displays information about your authentication state on each known GitHub host.
```

Comment:
1. **Should `Short` be consistent with first line of `Long`?**

   You'll notice that most / many Cobra commands even in `gh` are consistent as `Short` description is used in `gh auth --help` and `Long` description is used in `gh auth status --help`.

2. **Can this be more concise?**

```suggestion
		Short: "Display active account and authentication state on each known GitHub host",
		Long: heredoc.Docf(`

			Display active account and authentication state on each known GitHub host.
```

Reply:
Sure, I'd never noticed that pattern before. Fine with me though!

Reply:
Pulled this in in another commit since you can't apply suggestions on deleted lines.

Reply:
Can't wait for that limitation to be fixed, deleted lines